content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -11,73 +11,117 @@ OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
 
 ## ASP.NET Core
 
-The following example demonstrates automatic and manual instrumentation via an
-ASP.NET Core app.
+The following example demonstrates library based and manual instrumentation via an ASP.NET Core app.
 
-First, install all the required packages:
+First, create your basic ASP.NET Core site:
+
+```shell
+dotnet new mvc
+```
+
+Next, Add the Core OpenTelemetry packages
 
 ```shell
 dotnet add package OpenTelemetry.Exporter.Console
-dotnet add package OpenTelemetry.Extensions.Hosting --prerelease
-dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
-dotnet add package OpenTelemetry.Instrumentation.Http --prerelease
-dotnet add package OpenTelemetry.Instrumentation.SqlClient --prerelease
+dotnet add package OpenTelemetry.Extensions.Hosting
 ```
 
-Note that the `--prerelease` flag is required for all instrumentation packages
-because they are all are pre-release.
+Now let's add the instrumentation packages for ASP.NET Core. This will give us some automatic spans for each HTTP request to our app.
 
-This will also install the `OpenTelemetry` package.
+```shell
+dotnet add package OpenTelemetry.Instrumentation.AspNetCore --prerelease
+```
 
-### Tracing
+_Note that as the Semantic Conventions for attribute names are not currently
+stable the instrumentation package is currently not in a released state. That
+doesn't mean that the functionality itself is not stable. This means that you
+need to use the `--prerelease` flag, or install a specific version of the
+package_
 
-Next, add [tracing](/docs/concepts/signals/traces/) via the `System.Diagnostics`
-API.
+### Setup
 
-Paste the following code into your `Program.cs` file:
+Next, we need to add OpenTelemetry to our Service Collection in `Program.cs` so
+that it's listening correctly.
```

Comment:
nit: "its listening correctly" - does not really make it clear.

Reply:
Fixed